src/Domain/StrahlkorperTransformations.cpp

```diff
@@ -62,26 +61,26 @@ void coords_to_different_frame(
 
     // If this doesn't have a value, then the point isn't in the domain which is
     // really bad.
-    ASSERT(block_logical_coords[s].has_value(),
-           "Source coordinates are not in the domain.");
+    if (not block_logical_coords[s].has_value()) {
+      ERROR("A point on the Strahlkorper in the "
```

Comment:
Since this will be run over every point of the Strahlkorper, maybe we should wrap all these `if`s in `UNLIKELY` so the compiler can do some optimizations

Reply:
I don't know if `UNLIKELY` actually does something, but I can add it